nemo/collections/asr/parts/submodules/ctc_decoding.py

```diff
@@ -254,6 +254,7 @@ def __init__(self, decoding_cfg, blank_id: int):
                 compute_timestamps=self.compute_timestamps,
                 preserve_frame_confidence=self.preserve_frame_confidence,
                 confidence_method_cfg=self.confidence_method_cfg,
+                batched_inference=self.cfg.greedy.get('batched_inference', False),
```

Comment:
Argument name is too generic, CTC inference code runs in batch mode (just not fully batched mode).

This can just be a different strategy altogether honestly so instead of "greedy" and "pyctcdecode" etc we can just call this "greedy_batch" algo similar to RNNT ?

If not, can you assign some other name than "batched_inference"

Reply:
Making a new strategy seems like the right call. I'm called it "greedy_vectorized".